neo4j/templates/neo4j-config.yaml

```diff
@@ -126,6 +126,8 @@ data:
   dbms.routing.default_router: "SERVER"
   dbms.routing.client_side.enforce_for_domains: "*.{{ .Values.clusterDomain }}"
   dbms.routing.enabled: "true"
+  dbms.cluster.discovery.version: "{{ default "V1_ONLY" .Values.discoveryVersion }}"
```

Comment:
I was wondering about the namespacing, and whether this should in fact be `.Values.<something>.discoveryVersion`, so I went looking at the current `values.yaml`, and I was wondering whether the version specifically should in fact be [here](https://github.com/yirensum/helm-charts/blob/feature/default_v1_discovery/neo4j/values.yaml#L311) as a straight config value?

Though that's probably harder to override, users might have to override the whole block.

So maybe just add a `discoveryVersion: V1_ONLY` to the `neo4j` section of that file, and remove the `default "V1_ONLY"` from here, so it's a mandatory setting, default value is in the `values.yaml`, and it's easy to override.

Reply:
Yes, that looks good.